### DISCUSSION TITLE
Data residency of GitHub Enterprise Cloud service

### DISCUSSION THREAD

**Discussion by JyrkiHei** [community/community#38657] [Enterprise]
Hi!
We are using GitHub Enterprise Cloud organization and users are really interested in the data residency. Is the servers and data stored in US or somewhere else? It seems that with the coming GitHub AE you can choose the data location, but what about the current Enterprise Cloud?

Thanks in advance!

**Comment by ossanna16** [comment]
Hi there @JyrkiHei and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by refs** [comment]
@ossanna16  Any answer to this question? :)

**Reply by michaelhofer** [reply]
@JyrkiHei did you ever get a response to this good question? :)
